share/spice/sports/nfl/games/sports_nfl_games.js

```diff
@@ -65,6 +65,11 @@
                     // fill boxscore with placeholder data
                     attrs.score = games.fillBoxscore(attrs.score, ops);
 
+                    // finish here if there is no possession data
+                    if (!attrs.score.possession) {
```

Comment:
Nice! Is it possible that `attr.scores` is even undefined? If so, maybe we should catch that here too.